src/rust/zh.manhuagui/res/source.json

```
@@ -0,0 +1,10 @@
+{
+	"info": {
+		"id": "zh.manhuagui",
+		"lang": "zh",
+		"name": "manhuagui",
```

this is how it's capitalized on the site
```suggestion
		"name": "ManHuaGui",
```